src/component/sheet.js

```diff
@@ -915,14 +934,15 @@ function sheetInitEvents() {
 }
 
 export default class Sheet {
-  constructor(targetEl, data) {
+  constructor(targetEl, data, datas) {
```

Comment:
@aleschneck What is the use of the difference between `data` and `datas`? I think we may benefit from a quick comment to explain here.

Reply:
this is why typescript was invented :)